docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -69,6 +69,15 @@ NOTE: If no Events or Functions are specified, then the Sentinel will monitor AL
 
 IMPORTANT: At this time, Sentinels can only monitor internal function calls on Mainnet, Ropsten, and Kovan.  If the internal call emits an event, consider monitoring for that event instead.  Events emitted from internal calls are detected on all networks.
 
+[[hedera-support]]
+=== Hedera Support
+
+Hedera mainnet and testnet are supported in Defender Sentinels. There exist some considerations unique to this network when configuring a sentinel:
+
+* The sentinel monitors a contract's EVM address (starts with "0x") not the Hedera contract ID. The EVM address can be found on a Hedera Explorer such as [HashScan](https://hashscan.io/)
+* There is no remote resource to obtain contract ABIs as of now, so they must be generated from the solidity code using a [solidity compiler](https://docs.soliditylang.org/en/latest/installing-solidity.html).
+* The only conditions that can be monitored for internal transactions (contract calling contract) on the Hedera network are events. Other conditions will not trigger sentinels.
```

Comment:
Note: internal transaction events will not be technically monitored until this is merged: https://github.com/OpenZeppelin/defender/pull/2727